server/handler.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"bullfrogkv/storage"
+	"errors"
+	"github.com/gin-gonic/gin"
+	"log"
+	"net/http"
+)
+
+type putReq struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
```

Comment:
`putReq` starts with a lowercase letter, why should `Key` and `Value` start with an uppercase letter?